### ISSUE TITLE
too big offset error in popdel profile

### ISSUE THREAD

**Issue by edg1983** [kehrlab/PopDel#10]
Hi,
I'm trying to test popdel v1.2.1 on a small batch of 5 WGS sample mapped to GRCh38 (30-60X mean coverage).
The command I used is simply:

`popdel profile -o profiles/sample1.profile sample1.bam`

For all samples the program performs parameters estimation and then crash with the following error:

```
terminate called after throwing an instance of 'seqann:ParseError'
what(): Too big offset in Window!
```

Can you advise on what can cause this problem?

**Comment by serosko** [comment]
Hi,
are you using the conda version of PopDel or did you build it yourself using make?
In the latter case, could you try to rebuild popdel in debug mode using
`
make debug
`
in the source code directory then try again and send me the output?
Is your testing data by any chance publicly available? In that case I could try to reproduce the crash myself.

**Comment by edg1983** [comment]
I'm working on internal data, so I cannot share it directly.
I've compiled popdel with GCC v8.3.0. Now I've rebuilt popdel with debug mode and re-started the analysis. I will update you on the results.

**Comment by edg1983** [comment]
An update on this. I've downloaded the last version (1.2.2) and tried again on the previous system and also another one both based on centOS and the same GCC version. The executable build process performed exactly the same apparently (at least the same log messages).
However, on system one popdel crash after analysis of chromosome one with the previously reported message while on the system two everything goes fine. Unfortunately, system one has restricted access and I cannot move BAM files in or out so I haven't run the test on the same BAM file and I'm not sure if the issue is due to the input data, but at least we know it does not strictly depend on popdel itself. The next step will be to create a singularity image of PopDel test this on system one with BAM files and then use this to analyze BAM files on system two. This will tell us if the issue arises from some libraries or is due to the BAM files. BTW, have you plan to provide a container image (either singularity or docker) for popdel?

**Comment by serosko** [comment]
Thank you for trying this out and I am looking forward to the outcome.
As for the container image:
We provide a conda image of PopDel and they also automatically build a docker image of the uploaded software and deploy them to quay.io.
I have never tried to run the docker images, but maybe this is what you are looking for:
https://quay.io/repository/biocontainers/popdel?tab=tags

**Comment by serosko** [comment]
Fixed in release [v1.4.0](https://github.com/kehrlab/PopDel/releases/tag/v1.4.0).
